index: move donation bar below introduction section

Thynix opened the pull request:
Having the donation section above the introduction section and download button makes for a confusing situation, especially for first-time visitors or those on mobile devices for whom only the donation bar may be visible.

![Top](https://i.imgur.com/Np6P1nT.png)
![Below](https://i.imgur.com/JdQwRct.png)

Thynix commented:
Ooh, good idea. I begrudgingly admit that PHP made this easier to do than I would have thought.

Thynix commented:
Though this looks like it should work it actually doesn't. I'm looking into why. During the build it outputs:

    PHP Notice:  page=latestnews file=/home/operhiem1/website/includes/latestnews.inc.php in /home/operhiem1/website/includes/common.inc.php on line 81
    PHP Notice:  Undefined variable: pages in /home/operhiem1/website/includes/common.inc.php on line 86

Having something that's actually practical to test with instead of this pile of PHP and Bash would be helpful, but that's for the rewrite if we ever get around to it.

Thynix commented:
I think I have discovered two things:

1. `include`-ing a nonexistent file fails silently.
2. All files under a language are expected to have a counterpart in `includes/$page.inc.php`.

Thynix commented:
Whoops. I was not looking at the test site.

Thynix commented:
It worked all along. Thanks! Deployed!

ArneBab commented:
no problem - thank you for testing this! I used ./make-pages, but could only test the generated sites with lynx.

I still think that this solution is a hack, since it entangles the languages and does not allow sorting entries from different sources by date, but it works and is extremly simple. We just need to take care not to try to make this elegant and clever, otherwise I fear that it would grow horrible. It even relies on putting non-english new entries into the index pages, with a notice to do that in the page itself to keep people from doing clever (but dangerous) things. But it works and it is about as low maintenance as we can make it without introducing complexity (which people would have to understand to maintain the pages).

And thank you for deploying it

Thynix committed:
index: move donation bar below introduction section

Having the donation section above the introduction section and download
button makes for a confusing situation, especially for first-time
visitors or those on mobile devices for whom only the donation bar may
be visible.

The CSS filename is incremented to avoid caching.
